pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/buffer/impl/InMemTransactionBufferReader.java

```diff
@@ -66,15 +66,15 @@ public InMemTransactionBufferReader(TxnID txnId, Iterator<Entry<Long, ByteBuf>>
         int i = 0;
         while (i < numEntries && entries.hasNext()) {
             Entry<Long, ByteBuf> entry = entries.next();
-            TransactionEntry txnEntry = new TransactionEntryImpl(
-                txnId,
-                entry.getKey(),
-                EntryImpl.create(-1L, -1L, entry.getValue()),
-                committedAtLedgerId,
-                committedAtEntryId,
-                -1
-            );
-            txnEntries.add(txnEntry);
+//            TransactionEntry txnEntry = new TransactionEntryImpl(
+//                txnId,
+//                entry.getKey(),
+//                EntryImpl.create(-1L, -1L, entry.getValue()),
+//                committedAtLedgerId,
+//                committedAtEntryId,
+//                -1
+//            );
+//            txnEntries.add(txnEntry);
```

Comment:
Why we should delete these lines?

Reply:
The class `TransactionEntryImpl` was removed. The InMemTransactionBufferReader is unnecessary.